crates/rattler_installs_packages/src/python_env/venv.rs

```diff
@@ -291,11 +291,28 @@ prompt = {}"#,
                 .expect("cannot get system python parent folder");
             for bin_name in python_bins.into_iter() {
                 let original_python_bin = original_python_bin_dir.join(bin_name);
+                let original_python_scripts = original_python_bin_dir
+                    .join("Lib/venv/scripts/nt")
+                    .join(bin_name);
+                let venv_python_bin = venv_bin.join(bin_name);
 
-                if original_python_bin.exists() {
-                    let venv_python_bin = venv_bin.join(bin_name);
-                    if !venv_python_bin.exists() {
-                        copy_file(venv_exe_path, &venv_python_bin)?;
+                if original_python_bin.exists() && !venv_python_bin.exists() {
+                    if original_python_scripts.exists() {
+                        copy_file(original_python_scripts, &venv_python_bin)?;
+                    } else {
+                        // edge case for when used python is built from source code
+                        // or we use venv from build
```

Comment:
Not sure what the last line of the comment means :)?

Reply:
Modified the comment to better express the intention